v2.000: added variable fonts

m4rc1e opened the pull request:
Long time no see :-)

Type Network have recently been working on this family. This PR contains updated static fonts and a new variable font.

I've been proofing the family thoughout its development and am confident they've done a good job.

Commit:
conversion Glyph src to RF 3.3 UFO

Commit:
made first half of char set compatible

Commit:
Regular and Bold

Compatibilized

Commit:
Corrections

Compatibility completed

Commit:
generate and organize

Commit:
resolve plist error

Commit:
generation trial OTFs, TTFs, VFs

Commit:
add build.sh and out reports for both masters

Commit:
update build script

Commit:
cleanup repo and rebuild

Commit:
remove old .glyphs file

Commit:
remove non-ascii chars from font info

Commit:
rebuild

Commit:
Merge pull request #2 from TypeNetwork/fea-ufo-conv2

Fea ufo conv2

Commit:
build

Commit:
comment out some of the ccmp code in features

Commit:
update CCMP features

Commit:
rebuild

m4rc1e committed:
requirements added

m4rc1e committed:
build: don't hint fonts

m4rc1e committed:
regenerated fonts

m4rc1e committed:
Merge pull request #4 from TypeNetwork/fea-resolve-errors

Fea resolve errors

Commit:
Compatibility fixes and creation of VF.ttf

Build successful and appears completed.

Commit:
Merge branch 'fea-ufo-conv' of https://github.com/TypeNetwork/DancingScript into fea-ufo-conv

Commit:
Merge pull request #5 from TypeNetwork/fea-ufo-conv

Fea ufo conv

Commit:
Remove duplicate file

pichotta commented:
@impallari 
The weight axis variable font for DancingScript is complete. See Issue #2  Will you accept this pull request to merge the [wght].ttf and the .ufo sources into the master?

aaronbell committed:
Updating to UFR and updating stat table

Updating to UFR and updating stat table

and adding intermediate weights.

aaronbell committed:
Create build.yaml

aaronbell committed:
Rebuild static fonts with autohinting

I made an error in my previous PR in disabling autohinting for the static instances. This PR corrects that. Fonts rebuilt.

Sorry for the trouble!

Commit:
Merge pull request #8 from aaronbell/master

Update to UFR

aaronbell committed:
Updating URL to googlefonts

aaronbell committed:
Update README.md

aaronbell committed:
Update README.md

aaronbell committed:
Final updates

RosaWagner committed:
Merge pull request #9 from aaronbell/master

Updating URLS to googlefonts

davelab6 commented:
@impallari this is what is live in gf, please merge and pick up from here

impallari commented:
Thanks!
